.github/workflows/dependency-review.yml

```
@@ -0,0 +1,14 @@
+name: 'Dependency Review'
+on: [pull_request]
```

```suggestion
name: Dependency Review
on: pull_request
```

tiny consistency tweak